packages/sanity/src/core/form/inputs/PortableText/_legacyDefaultParts/__tests__/BlockExtras.test.tsx

```diff
@@ -2,8 +2,8 @@ import {fireEvent, waitFor} from '@testing-library/react'
 import React from 'react'
 import {PortableTextMarker} from '../../../../types'
 // import {renderInput} from '../../../../test/renderInput'
-import {PortableTextInput, PortableTextInputProps} from '../../PortableTextInput'
-import {portableTextType} from './schema'
+// import {PortableTextInput, PortableTextInputProps} from '../../PortableTextInput'
+// import {portableTextType} from './schema'
```

Comment:
can these be removed?

Reply:
Yes, probably! But there were a few things commented out in that file that used these types, so I thought they might as well be left commented out

Reply:
Ah, I see, makes sense to keep them, then. Thnx for clarifying!